docs/Symbols.md

```diff
@@ -64,6 +64,8 @@ The order of the expression item are important, because each expression are appl
 
 NOTE: when you change `xml.symbols.filters` in `settings.json`, the Outline is not refreshed automatically (see vscode [vscode issue 108722](https://github.com/microsoft/vscode/issues/108722)). You must refresh at hand the Outline by updating the XML file or closing/opening the XML file.
 
+NOTE: when you define your own patterns in `settings.json`, it might be necessary to define the glob pattern that crosses directory boundaries `**` for the matching to occur. ([vscode-xml issue 684](https://github.com/redhat-developer/vscode-xml/issues/684)). For example, when applying this to xml and bpml files this might be necessary: `"pattern": "**/*.{bpml,xml}",` 
```

Comment:
"(See `[vscode-xml issue 684](https://github.com/redhat-developer/vscode-xml/issues/684)` for more detail.)."

Reply:
I removed the ( ) since it no longer made sense to have them around a full sentence.